snuba/query/schema.py

```diff
@@ -145,31 +146,54 @@
         # Conditions at the second level are ORed together.
         # eg: [(a, =, 1), (b, =, 2)] => "a = 1 AND b = 2"
         # eg: [(a, =, 1), [(b, =, 2), (c, =, 3)]] => "a = 1 AND (b = 2 OR c = 3)"
-        "condition": {
+        "condition_lhs": {
+            "anyOf": [
+                {"$ref": "#/definitions/column_name"},
+                {"$ref": "#/definitions/nested_expr"},
+            ],
+        },
+        "binary_condition": {
             "type": "array",
             "items": [
+                {"$ref": "#/definitions/condition_lhs"},
                 {
+                    # Operator
+                    "type": "string",
+                    "enum": BINARY_OPERATORS,
+                },
+                {
+                    # Literal
                     "anyOf": [
-                        {"$ref": "#/definitions/column_name"},
-                        {"$ref": "#/definitions/nested_expr"},
+                        {"type": ["string", "number"]},
```

Comment:
Hmm, wasn't null allowed here before?

Reply:
oh you are right. `a = NULL` could still be a thing